app/views/sessions/new.html.erb

```diff
@@ -3,21 +3,21 @@
 
 <div class="container">
   <div class="row justify-content-center">
-    <div class= "col-10">
+    <div class= "col-md-10 col-sm-12">
```

Comment:
Not really related to this PR, but i feel like using `col` classes to decrease the container width on all pages (as far as i can tell) is not the right tool for the job. Not completely wrong, either, but if you want to improve this, you could check out the bootstrap doc, on [how to theme bootstrap](https://getbootstrap.com/docs/4.0/getting-started/theming/) and [specifically the container width](https://getbootstrap.com/docs/4.0/layout/grid/#grid-tiers).

Reply:
Ah I see what you mean, this is a good point. If, however, `col` is good enough, then I would prefer to leave it like this for now :smiley_cat: